File: Beecrowd/1134 Tipo de Combustível/main.py
al = 0
gas = 0
diesel = 0
X = 1
	
while X != 4:
  X = int(input())
  if X == 1:
    al += 1
  if X == 2:
    gas +=1
  if X == 3:
    diesel += 1
				
print("MUITO OBRIGADO")
print("Alcool: {}".format(al))
print("Gasolina: {}".format(gas))
print("Diesel: {}".format(diesel))

File: Beecrowd/1134 Tipo de Combustível/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int X, i, al = 0, gas = 0, diesel = 0;
	
	do{
			cin >> X;
			
			if(X == 1)
				al++;
			if(X == 2)
				gas++;
			if(X == 3)
				diesel++;
				
	}while(X != 4);
		cout << "MUITO OBRIGADO" << endl;
		cout << "Alcool: " << al << endl;
		cout << "Gasolina: "<< gas<< endl;
		cout << "Diesel: "<< diesel << endl;
 
    return 0;
}